lib/rubygems/version.rb

```diff
@@ -279,7 +279,10 @@ def encode_with coder # :nodoc:
   # A version is considered a prerelease if it contains a letter.
 
   def prerelease?
-    @prerelease ||= !!(@version =~ /[a-zA-Z]/)
+    unless defined?(@prerelease)
```

Comment:
> it's his call

Very true, I was just hoping for a little more discussion back then, but I guess it's not to be. Anyway, I changed the code.